README.md

```diff
@@ -131,6 +132,29 @@ archive { '/tmp/test100k.db':
 }
 ```
 
+### Puppet URL
+
+Currently there isn't support for using ```puppet:\\``` style url in the source parameter.  It is still possible, but it would be done in two steps:
```

Comment:
Wrong slashes puppet:///, and please note do not enable  `cleanup=>true` which is not idempotent with this work around.